engine/src/ide.cpp

```diff
@@ -535,7 +535,7 @@ void MCIdeScriptConfigure::exec_ctxt(MCExecContext &ctxt)
                 MCAutoStringRef t_attr_value;
 
                 MCStringFormat(t_attr_key_string, "%s attributes", s_script_keywords[t_keyword]);
-                MCNameCreate(t_attr_key_string, &t_attr_key);
+                MCNameCreateAndRelease(t_attr_key_string, &t_attr_key);
```

Comment:
While you're here, these should probably be marked `/* UNCHECKED */`

Reply:
Done!